safe/gui/widgets/test/field_mapping_tab_example.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+"""Example usage of field mapping widget.."""
```

Comment:
It's not a test file, but an example to see how it looks. I put it in the test directory since I want it to be removed when we do the packaging.